### ISSUE TITLE
How can I create a local ROBERT server and applications circle

### ISSUE THREAD

**Issue by ilkane** [ROBERT-proximity-tracing/documents#75]
Hi,

I am trying to install the robert-server in my local computer to run some evalutions. I download from: https://gitlab.inria.fr/stopcovid19/robert-server/-/tree/develop/
There is a docker-compose file. I firstly "maven install" the robert-server-batch, robert-server-ws-rest and robert-crypto-grpc-server so that the app.jar files can be created. Everything looks okay so far.
Then, I run the "docker-compose up", all the services start to run but (robert-server-batch and robert-server-flyway ) exits after some operations.
![Screenshot 2022-03-31 at 11 14 57](https://user-images.githubusercontent.com/6386267/161021277-25db6835-7edc-4081-9596-c072d3277360.png)

Robert-server-flyway error message is : 
ERROR: Found non-empty schema(s) "public" but no schema history table. Use baseline() or set baselineOnMigrate to true to initialize the schema history table. robert-server_flyway_1 exited with code 1

Robert-server-batch error message is:
Closed connection [connectionId{localValue:5, serverValue:15}] to mongo:27017 because the pool has been closed. robert-server_batch_1 exited with code 0

Also, the robert-server-ws-rest is running on 0.0.0.0:8086 but I don't know how I should call the services in this server for example, the mobile application wants to upload the collected EBIDs to the server. 

In short, is there any implementation document that you can share with me or at least could you please give me some directions in here so that I can start experimenting around?

Many thanks beforehand.
